crates/vm/levm/src/precompiles.rs

```diff
@@ -421,27 +428,20 @@ pub fn ecadd(
         .concat();
         Ok(Bytes::from(res))
     } else if second_point_is_zero {
-        // If second point is zero, return is first point
+        // If second point is zero, response is first point
```

Comment:
ditto

Reply:
Done in [Use main's version in ecmul and ecadd](https://github.com/lambdaclass/ethrex/pull/1560/commits/cd4a769adbcfd1d214ddbf905053e1ff3d74408d)